### PR TITLE
Add uniquekey and addchainid to the sdk

### PR DESCRIPTION
**Pull request by aomerk**

### CODE REVIEW

File: python-sdk/src/forta_agent/finding.py [hunk 1]
```diff
@@ -38,6 +38,7 @@ def __init__(self, dict):
         self.metadata = dict.get('metadata')
         self.addresses = dict.get('addresses')
         self.labels = list(map(lambda l: l if isinstance(l, Label) else Label(l), dict.get('labels', [])))
+        self.unique_key = dict.get('unique_key')
```
Comment (haseebrabbani) [inline python-sdk/src/forta_agent/finding.py:41]: do we need to add `chain_id` here as well?

File: sdk/finding.ts [hunk 2]
```diff
@@ -67,7 +78,9 @@ export class Finding {
     type,
     metadata = {},
     addresses = [],
-    labels = []
+    labels = [],
+    uniqueKey = '',
+    chainId = -1,
```
Comment (haseebrabbani) [inline sdk/finding.ts:83]: not sure we should be setting these default values unless we want them being set in the backend even when a user does not specify them. i think we should leave them as undefined

File: sdk/finding.ts [hunk 3]
```diff
@@ -78,6 +91,9 @@ export class Finding {
     // TODO assert metadata keys and values are strings
 
     labels = labels.map(l => l instanceof Label ? l : Label.fromObject(l))
-    return new Finding(name, description, alertId, protocol, severity, type, metadata, addresses, labels)
+    const sources = {chainSource: {chainId: chainId}}
```
Comment (haseebrabbani) [inline sdk/finding.ts:94]: i user does not specify chainId input, should this be undefined as well?

File: python-sdk/src/forta_agent/finding.py (hunk 1 above)
Comment (haseebrabbani) [reply python-sdk/src/forta_agent/finding.py:41]: also should `sources` be set somewhere in the constructor?

File: sdk/finding.ts (hunk 3 above)
Comment (aomerk) [reply sdk/finding.ts:94]: yes

**Review by stevenlanders — APPROVED** [review]

**Pull request merged**